Apps/Contoso.Android.Puppet/Properties/AndroidManifest.xml

```diff
@@ -1,5 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.microsoft.appcenter.xamarin.puppet" android:versionCode="42" android:versionName="1.0.2-SNAPSHOT" android:installLocation="auto">
 	<uses-sdk android:minSdkVersion="16" />
-	<application android:label="SXPuppet" android:icon="@drawable/Icon" android:theme="@style/PuppetTheme" />
+	<application android:label="SXPuppet" android:icon="@drawable/Icon" android:theme="@style/PuppetTheme">
+		<receiver android:name="com.google.firebase.iid.FirebaseInstanceIdInternalReceiver" android:exported="false" />
```

Comment:
you also need to remove those from the forms.puppet one as we removed firebase in that one.
When the registration intent will be fired in the non firebase app it will try to launch FirebaseInstanceIdReceiver and class not found.

Reply:
Fixed in https://github.com/Microsoft/AppCenter-SDK-DotNet/pull/508/commits/8c218f6c059bb1f7150729fa143199c9383452d3